### ISSUE TITLE
overwrite and overlay

### ISSUE THREAD

**Issue by fredlebel** [jmillikin/haskell-ncurses#4]
I find I need these two functions.  In my fork I implemented them as `overwriteFrom` and `overlayFrom :: Update()` but the names can change.

Thanks

**Comment by fredlebel** [comment]
`copywin` would also be good to have.  From my fork as `copyFrom`.

**Comment by jmillikin** [comment]
Could you try out commit 05ed6f93195c31e6afc7d5eab62a4be1d88bf913 and let me know if that works for you? If yes, then I'll push a new release to Hackage.

``` haskell
data OverlayMode = OverlayMerge | OverlayReplace
overlay :: Window -> OverlayMode -> Update ()
copyWindow :: Window -> OverlayMode -> ... -> Update ()
```

**Comment by fredlebel** [comment]
Works as expected.  Thanks.

**Comment by jmillikin** [comment]
Released in version 0.2.15.
